CIP-0015/README.md

```diff
@@ -1,45 +1,52 @@
 ---
 CIP: 15
-Title: Catalyst Registration Transaction Metadata Format
-Authors: Sebastien Guillemot <sebastien@emurgo.io>, Rinor Hoxha <rinor.hoxha@iohk.io>, Mikhail Zabaluev <mikhail.zabaluev@iohk.io>
-Comments-URI: https://forum.cardano.org/t/cip-catalyst-registration-metadata-format/44038
+Title: Registration Transaction Metadata Format
+Category: Metadata
 Status: Active
-Type: Standards
+Authors:
+    - Sebastien Guillemot <sebastien@dcspark.io>, 
+    - Rinor Hoxha <rinor.hoxha@iohk.io>, 
+    - Mikhail Zabaluev <mikhail.zabaluev@iohk.io>
+Implementors:
+    - Daedalus <https://daedaluswallet.io/>,
+    - DcSpark <https://www.dcspark.io/>,
+    - Eternl <https://eternl.io/>,
+    - Flint <https://flint-wallet.com/>,
+    - Project Catalyst <https://projectcatalyst.io/>,
+    - Typhon <https://typhonwallet.io/>,
+    - Yoroi <https://yoroi-wallet.com/>
+Discussions:
+    - https://forum.cardano.org/t/cip-catalyst-registration-metadata-format/44038
+    - https://github.com/cardano-foundation/cips/pulls/58
 Created: 2020-01-05
 License: CC-BY-4.0
 ---
 
 ## Abstract
 
-Cardano uses a sidechain for its treasury system. One needs to "register" to participate on this sidechain by submitting a registration transaction on the mainnet chain. This CIP details the registration transaction format.
+This CIP details a transaction metadata format, used by Cardano's [Project Catalyst](https://projectcatalyst.io) for voter registrations.
 
-## Motivation
+## Motivation: why is this CIP necessary?
 
-Cardano uses a sidechain for its treasury system ("Catalyst"). One of the desirable properties of this sidechain is that even if its safety is compromised, it doesn't cause loss of funds on the main Cardano chain. To achieve this, instead of using your wallet's recovery phrase on the sidechain, we need to use a brand new "voting key".
+Project Catalyst uses a sidechain for it's voting system.
+One of the desirable properties of this sidechain is that even if its' safety is compromised, it doesn't cause a loss of funds on Cardano mainnet. 
+To achieve this, instead of using Cardano wallets' recovery phrase on the sidechain, we introduce the "voting key".
 
-However, since 1 ADA = 1 vote, a user needs to associate their mainnet ADA to their new voting key. This can be achieved through a registration transaction.
+However, since Catalyst uses stake-based voting, a user needs to associate their mainnet Ada to their voting key. 
+This can be achieved through a voter registration transaction.
 
-We therefore need a registration transaction that serves three purposes:
+We therefore need a voter registration transaction that serves three purposes:
 
-1. Registers a "voting key" to be included in the sidechain
-2. Associates mainnet ADA to this voting key
-3. Declare an address to receive Catalyst rewards
+1. Registers a voting key to be included in the sidechain
+2. Associates mainnet Ada to this voting key
```

Comment:
```suggestion
2. Associates mainnet ada to this voting key
```
Yes this was incorrectly full-caps before, but they're non-specific ada so the term isn't capitalised at all 😇